Atos20/reddit_clone
# Enviroment set up

### PR Template:

#### What’s this PR do?  
 - installs all required dependencies to use Typescript 
 - Sets up the Mikron environment to migrate table using the CLI
 - Sets a basic Express server  
#### Where should the reviewer start?  
 `index.ts`

#### How should this be manually tested?  
if you want to clone this repo, hit the `clone` button and make sure to run `yarn install`
 - run `yarn watch` to listen for all changes
-  run `yarn dev` to list for the changes applied to `index.js`
- Server should be listening on port 5000
- The screenshot below shows what should appear on the page after visiting `localhost:5000`
#### Any background context you want to provide?  

This is a project I am working on where I practice setting up the development environment without the use of any boilerplate. This will enable me to practice and understand what each library is aiming to accomplish in the project. 

My goal is to have a deeper understanding of the technologies I am using as well as keep my CLI skills sharp.

#### What are the relevant tickets?  
 n/a
#### Screenshots (if appropriate) 
 ![Screen Shot 2021-02-10 at 9 44 46 PM](https://user-images.githubusercontent.com/56229864/107603201-37a45280-6be9-11eb-9542-3e1d7e8190c4.png)
